### PR TITLE
Dealing with the .env and treating responses separately issues 

### LINKED ISSUE #1: Use .env
Instead of some arbitrary file name like "disco_link.txt" use .env format as it is a widely accepted format and practice.

Additionally use python's way to load .env instead of manual parsing.

### LINKED ISSUE #2: More flexible way of checking the message and reacting
Currently the codebase is basic enough to have regex check but to be future proof and testable it would be nice that "good morning" to be separated.

### PR DESCRIPTION
**Pull request by JordanTK**
Created .env for storing environment variables as requested. Also moved the response functionality in a separate file, so now every response is a class, it is added to a list inside the bot functional file and each response should have a process method dealing with any recursive reactions t authors message and respond method to respond to a message in the case of GoodMorning with a reaction.

### CODE REVIEW

File: bot.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
-import discord,re
-with open("disco_link.txt", "r") as f:
-    t = f.readline()
+import discord,os,responses
```
Comment (yumerov) [inline bot.py:1]: where are these dependencies described as installable? I guess we have to use requirements.txt method.

File: responses.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+            return
+        
+    def dobro(self, message):
+        return "добро" in str(message.content).lower()
+    
+    def utro(self,message):
+        return "утро" in str(message.content).lower()
+    
+    def respond(self,message):
+        if self.dobro(message) and self.utro(message):
```
Comment (yumerov) [inline responses.py]: According to this logic, "утро добро" is a valid case and I do not mind.

File: responses.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class GoodMorning:
+    def process(self, message,client):
+        if message.author == client.user:
```
Comment (yumerov) [inline responses.py]: Are we sure that it is going to stop the loop?

Comment (JordanTK) [reply responses.py]: Should do the work. There is no messaging anyways so i am removing it for now it is not needed.

File: responses.py (hunk 2 above)
Comment (JordanTK) [reply responses.py]: I do. Will make changes accordingly

File: bot.py (hunk 1 above)
Comment (JordanTK) [reply bot.py:1]: Got it, much appreciated!

**Pull request merged**